### DISCUSSION TITLE
有人用day planner插件吗？timeline视图出不来，求助

### DISCUSSION THREAD

**Discussion by lillian-who** [obsidianzh/forum#87] [Q&A常见问题]
今天发现obsidian开放了好多插件，赶紧安装了日历插件和day planner，就是看中了那个时间线看上去很炫酷。复制了官方的demo，结果右边出现不了时间线，有人知道是什么情况吗？

**Comment by lillian-who** [comment]
已解决。

原来要打开命令面板，然后输入：`Show the Day Planner Timeline`命令


![image](https://user-images.githubusercontent.com/20553685/121626692-7ddec500-caa8-11eb-9038-be2983e64e4f.png)

**Comment by lillian-who** [comment]
放个效果图：
![image](https://user-images.githubusercontent.com/20553685/121626893-dca43e80-caa8-11eb-8190-07dcb4dfe111.png)

**Comment by fred1357944** [comment]
Cool!

**Comment by fred1357944** [comment]
![07B21AC7-CA88-4DEB-8696-D37B8CC5FE75](https://user-images.githubusercontent.com/34299763/131233725-2debf57e-0e49-4b84-98d6-84529941d555.jpeg)
也可以結合mermaid 製作與連動自己的day planner喔
